includes/functions/updates/llms-functions-updates-600.php

```diff
@@ -208,15 +215,19 @@ function _migrate_awards( $type ) {
 	// Don't trigger save hooks.
 	remove_action( "save_post_llms_my_{$type}", array( 'LLMS_Controller_Awards', 'on_save' ), 20 );
 
-	$legacy_option_added = false;
+	$legacy_certificates_option_added = false;
+	$legacy_achievements_option_added = false;
```

Comment:
the whole method runs `once` per `$type` right?
So we don't need to keep track of both these _additions_ or differentiate between achievements and certificates here. Not anymore. So:
```suggestion
	$legacy_option_added = false;
```

Reply:
Good catch.